.github/workflows/check-approvals.yml

```diff
@@ -7,10 +7,7 @@ on:
       - unlabeled
       - opened
       - reopened
-      - synchronize
-      - ready_for_review
   pull_request_review:
-    types: [submitted, edited, dismissed]
```

Comment:
So this PR just removes a few things? It looks like `pull_request_review:` has no value now - is that okay?

```yaml
on:
  pull_request:
    types:
      - labeled
      - unlabeled
      - opened
      - reopened
  pull_request_review:
```

Reply:
Correct, it just needs to have a colon after it for all of the accepted types for that trigger to be used. I then also removed some redundant types that I was using for testing of the original PR.